lxd/network/driver_bridge.go

```diff
@@ -3632,6 +3632,18 @@ func (n *bridge) Leases(projectName string, clientType request.ClientType) ([]ap
 			return nil, err
 		}
 
+		leasesCh := make(chan api.NetworkLease)
+
+		var wg sync.WaitGroup
+		wg.Add(1)
```

Comment:
Sorry, just noticed this now. Since we are dealing with only one goroutine, would using a channel instead of a waitGroup be more appropriate?